pkgs/by-name/ui/uiua386/package.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, stdenvNoCC, fetchFromGitHub }:
```

Comment:
```suggestion
{ lib, stdenvNoCC, uiua }:
```

Reply:
yeah, I just had to rebase the latest master.